### PR TITLE
Correct resource quota statement

### PR DESCRIPTION
**Pull request by tengqm**
Closes: #6947

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6964)

### CODE REVIEW

File: docs/concepts/policy/resource-quotas.md [hunk 1]
```diff
@@ -46,9 +46,8 @@ Resource Quota support is enabled by default for many Kubernetes distributions.
 enabled when the apiserver `--admission-control=` flag has `ResourceQuota` as
 one of its arguments.
 
-Resource Quota is enforced in a particular namespace when there is a
-`ResourceQuota` object in that namespace.  There should be at most one
-`ResourceQuota` object in a namespace.
+Resource Quota is enforced in a particular namespace when there is at least one
+`ResourceQuota` object in that namespace.
```
Comment (zhangxiaoyu-zidif) [inline docs/concepts/policy/resource-quotas.md]: should we make more than one resourcequota in one namespaces, if there is conflict...? so I am not sure about it.

Comment (tengqm) [reply docs/concepts/policy/resource-quotas.md]: I have tried it using v1.9 and it seems okay. I didn't spend time on reasoning the potential conflicts though.

**Comment by heckj** [comment]
@vishh, @random-liu, @tallclair - could one of you help clarify? We need a technical accuracy review to better update the docs...

**Comment by zacharysarah** [comment]
Bumping for technical review.

/assign @vishh

**Comment by zacharysarah** [comment]
Bumping for technical review.

@tallclair 👋 Would you please take a quick look and `/lgtm` if this change works for you?

**Comment by zacharysarah** [comment]
Bumping for technical review.

/assign @tallclair 
/assign @Random-Liu

File: docs/concepts/policy/resource-quotas.md [hunk 2]
```diff
@@ -46,8 +46,8 @@ Resource quota support is enabled by default for many Kubernetes distributions.
 enabled when the apiserver `--admission-control=` flag has `ResourceQuota` as
 one of its arguments.
 
-A resource quota is enforced in a particular namespace when there is a
-`ResourceQuota` in that namespace.
+Resource Quota is enforced in a particular namespace when there is at least one
+`ResourceQuota` object in that namespace.
```
Comment (tallclair) [inline docs/concepts/policy/resource-quotas.md]: Whether this is default allow or default deny depends on the configuration ([`LimitedResources`](https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/resourcequota/apis/resourcequota/types.go#L29))

**Comment by heckj** [comment]
@tengqm please update this PR to address @tallclair's feedback

**Comment by tengqm** [comment]
@heckj Still need some time to work on this. The original idea was a quick fix to the issue raised. By experiment, we found that there are no restrictions on whether two quota objects are created for the same resource. That was the intent of this fix. @tallclair was mentioning an internal LimitedResources configuration not exposed from API interface. I did spent some time reading the source code. The quota behavior is not that straightforward for a quick fix. Let me try allocate some bandwidth on this.

**Comment by zacharysarah** [comment]
@tengqm 👋 We've migrated the site's framework to Hugo. Please re-open this PR with changes to files in their post-migration location (`/content/en/docs/...`).

**Pull request closed without merging**